braidpool_spec.md

```diff
@@ -499,6 +754,17 @@ $n-t$ nodes successfully delete these shares and keys, and the RCA and UHPO
 transactions are distributed to all nodes, it then becomes impossible to spend
 the aggregated Braidpool funds in any other way.
 
+[kp]
+
+1. What is RCA?
+
+2. Whoa. We trusting miners to behave themselves and delete the key shares?
+
+3. Before we get into the details, we should first talk about the protocol that
+   constitutes the ceremony.
+
+[/kp]
+
```

Comment:
RCA = Rolling Commitment Authorization -- the output representing the sum of pool funds, which is then spent by the UHPO transaction.

Deleting key shares -- this is possible with Verifiable Secret Sharing. As long as $n-t$ delete shares, the remaining non-deleted shares are not enough ($t$) to reconstruct the secret, and the secret is gone forever. I don't think this is actually necessary -- it has to do with FROST using VSS for shares but as long as after the fact, old shares cannot sign new transactions, I don't think deleting shares is needed.